src/environments/array.js

```diff
@@ -351,8 +352,9 @@ const htmlBuilder: HtmlBuilder<"array"> = function(group, options) {
 
     // Add \hline(s), if any.
     if (hlines.length > 0) {
-        const line = buildCommon.makeLineSpan("hline", options, 0.05);
-        const dashes = buildCommon.makeLineSpan("hdashline", options, 0.05);
```

Comment:
This change is what caused the change the screenshots.  I assume these should've been 0.04 before?

Reply:
They were 0.05 before. When I wrote `\hline`, I followed the example of the {array} vertical lines, which were also 0.05. But that isn't what LaTeX does.